2-mandatory/4-restaurant.js

```diff
@@ -19,10 +19,33 @@ const MENU = {
   falafel: 7.25,
 };
 
+function orderBurger(balance) {
+  if (balance - MENU.burger >= 0) {
+    return balance - MENU.burger >= 0
+  }
+}
+
+return balance;
```

Comment:
it looks like your return statements on this file are misplaced, they should be in the function definitions, similar to the ones you've written on `3-shopping-list.js`